### PR TITLE
add migration script to generate default sass var

### PR DESCRIPTION
**Pull request by melvinsoft**

### CODE REVIEW

**Comment by melvinsoft** [comment]
[(2) HTML Certs production push](https://trello.com/c/DeR7iHqJ/1856-2-html-certs-production-push)

File: openedx/core/djangoapps/appsembler/sites/migrations/0002_auto_20180306_0747.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import json
+
+from django.db import migrations, models
+from openedx.core.djangoapps.site_configuration.models import SiteConfiguration
```
Comment (bezidejni) [inline openedx/core/djangoapps/appsembler/sites/migrations/0002_auto_20180306_0747.py]: I don't think you can import models directly, if you look at the [migration docs](https://docs.djangoproject.com/en/1.11/topics/migrations/#data-migrations), you'll see that they use `apps.get_model()` to import the model. Also, look at the section below about importing models from other apps to make sure you set a dependency on the `site_configuration` app.

**Comment by melvinsoft** [comment]
@bezidejni Thanks! all changes addressed! Please take another look.

**Comment by melvinsoft** [comment]
@bezidejni Thanks!

**Pull request merged**